### ISSUE TITLE
no rfid tag detected

### ISSUE THREAD

**Issue by sroesslhuemer** [blinzelaffe/roseguarden#11]
welche moeglichkeiten habe ich rauszufinden warum kein tag erkannt wird < keine reaktion obwohl alles lt. schema verdrahtet

**Comment by mdrobisch** [comment]
Hallo Siegi,
ich hatte dir am 9.11 geschrieben. Ist die Mail evt. im Spam gelandet?

SG
Marcus

**Comment by sroesslhuemer** [comment]
hallo marcus,

...konnte inzwischen das problem orten und lösen (siehe unten)
- habe jetzt 2 voneinander unabhängige systeme - jetzt meine frage - was muss ich tun, um ein system zu haben ?

weiters - unter statistics sagt er mir dass evtl. ein parameter nicht gesetzt ist (umgebungsvariable?)

danke für die infos 

#######################################
Make sure the SPI Interface is enabled:
Edit the file /boot/config.txt and add the following two lines:
device_tree_param=spi=on
dtoverlay=spi-bcm2708
Reboot the Pi and check if the SPI Interface software is running:
$ dmesg | grep spi
You should see something like:
[    8.577659] bcm2708_spi 20204000.spi: master is unqueued, this is deprecated
[    8.578491] bcm2708_spi 20204000.spi: SPI Controller at 0x20204000 (irq 80)
You can also check:
$ lsmod
which should list:
spi_bcm2708             5429  0_

**Comment by sroesslhuemer** [comment]
hallo marcus,

...konnte inzwischen das problem orten und lösen (siehe unten)

```
habe jetzt 2 voneinander unabhängige systeme - jetzt meine frage - was muss ich tun, um ein system zu haben ?
```

weiters - unter statistics sagt er mir dass evtl. ein parameter nicht gesetzt ist (umgebungsvariable?)

danke für die infos

#######################################
Make sure the SPI Interface is enabled:
Edit the file /boot/config.txt and add the following two lines:
device_tree_param=spi=on
dtoverlay=spi-bcm2708
Reboot the Pi and check if the SPI Interface software is running:
$ dmesg | grep spi
You should see something like:
[ 8.577659] bcm2708_spi 20204000.spi: master is unqueued, this is deprecated
[ 8.578491] bcm2708_spi 20204000.spi: SPI Controller at 0x20204000 (irq 80)
You can also check:
$ lsmod
which should list:
spi_bcm2708 5429 0_

**Comment by mdrobisch** [comment]
Hallo Siegi,
Entschuldige die späte Antwort. Deine Nachricht ist leider nicht bei mir angekommen.
Läuft dein System mittlerweile oder gab es noch Probleme?

SG
Marcus

PS: Mit dem neuen Release 1.0.0 kommt ein Installer, der dir die Konfiguration abnimmt.

**Comment by sroesslhuemer** [comment]
Hallo,

 

....nein läuft nur im single modus – wenn ich 2 zusammenhänge geht’s leider nicht

...wann kommt das neue release ? – und hilft es mir wenn ich mehrere schlösser zusammenhänge ?

 

Lg

siegi

 

Von: mdrobisch [mailto:notifications@github.com] 
Gesendet: Mittwoch, 19. April 2017 17:30
An: mdrobisch/roseguarden
Cc: sroesslhuemer; State change
Betreff: Re: [mdrobisch/roseguarden] no rfid tag detected (#11)

 

Hallo Siegi,
Entschuldige die späte Antwort. Deine Nachricht ist leider nicht bei mir angekommen.
Läuft dein System mittlerweile oder gab es noch Probleme?

SG
Marcus

PS: Mit dem neuen Release 1.0.0 kommt ein Installer, der dir die Konfiguration abnimmt.

**Comment by mdrobisch** [comment]
Ja, das wird damit auch einfacher. Die IP des masters angeben und fertig. Ich hab da schon eine Idee, wie das funktionieren kann. Der Installer ist relativ einfach zu bedienen. Das 1.0.0 Release plane ich Ende Juni.

**Comment by timo5500** [comment]
Hi,

ich habe das selbe problem ich kann aber mit den tests scripts die tags lesen aber das webinterface zeigt immer an No rfid-tag detected

**Comment by Sico93** [comment]
@mdrobisch Ich habe leider auch noch das selbe Problem im Webinterface. Die Testskripte funktionieren alle, ich verstehe aber einfach nicht wieso die Website einfach keine Tags erkennt. 
Ich bin scheinbar auch einfach nicht in der Lage herauszufinden welches Skript da abgerufen wird :( Ich würde mich über eine Antwort freuen, auch wenn das Projekt scheinbar tot gelaufen ist :/

**Comment by mdrobisch** [comment]
Hallo Zusammen,
aufgrund tiefgreifender Probleme in der Architektur haben wir beschlossen, Roseguarden vollständig neu aufzusetzen. Wir arbeiten in einer Gruppe von 3 Personen am Nachfolger und werden keine Zeit mehr in die Pfelge der alten Version stecken. Sorry hierfür.
Die Neuauflage ist unter https://gitlab.com/roseguarden/roseguarden zu finden.

**Comment by sroesslhuemer** [comment]
hallo,
...was heisst in diesem fall neu aufgesetzt ? - ist das eine kpl. neue hardware oder kann ich die alte hardware verwenden ?
lg

**Comment by mdrobisch** [comment]
Hallo sroesslhuemer,
Kurz zusammengefasst soll es folgende Veränderungen geben.
- Verwaltung auf einen Server (nicht nur lokal), damit auch sichere Verwaltung über das Internet möglich wird (lokales Intranet über LAN wird ebenfalls weiterhin möglich sein). Der Prototyp ist hier zu finden https://roseguarden.fabba.space/dashboard
- Einen besseren Reader mit höherer Reichweite https://www.olimex.com/Products/Modules/RFID/MOD-RFID1356MIFARE/
- Optionales Display zur Anzeige des Status, vermutlich auch mit Zugang über 1-mal-Pins https://www.olimex.com/Products/Modules/LCD/MOD-LCD2-8RTP/open-source-hardware
- Unabhängigkeit von Lesegerät und z.B. Türöffner (Lesegerät in einem anderen Raum als der Türöffner) https://www.olimex.com/Products/IoT/ESP32/ESP32-EVB/open-source-hardware

Die alte Hardware kann ggf. wiederverwendet werden. Ich kann dabei gern unterstützen. Wir haben Adapterboard entwickelt, die kann ich gerne rausgeben.

Beste Grüße



-
